### PR TITLE
Add composer.json

### PR DESCRIPTION
**Pull request by QWp6t**

### CODE REVIEW

**Comment by QWp6t** [comment]
For anyone who wants to test this PR, merge the following into your composer.json file...

```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/qwp6t/wp-api"
        }
    ],
    "require": {
        "wp-api/wp-api": "dev-add-composer"
    }
```

**Comment by retlehs** [comment]
:+1: this closes #310 as well

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "name": "wp-api/wp-api",
+    "type": "wordpress-plugin",
+    "description": "Access your WordPress site's data through an easy-to-use HTTP REST API.",
+    "homepage": "http://wp-api.org/",
+    "license": "GPL2+",
+    "authors": [
+        {
+            "name": "Ryan McCue",
+            "email": "me+github@ryanmccue.info",
+            "homepage": "http://ryanmccue.info/"
```
Comment (danielbachhuber) [inline composer.json]: Can we make this "WP-API Team"?

**Comment by QWp6t** [comment]
Changed author to WP-API Team

@danielbachhuber

**Pull request merged**